README.md

```diff
@@ -63,6 +71,8 @@ locheck [...]
 
 Other install methods may be added upon request as we discover people's needs. This project is very new and setting up new installation methods takes time.
 
+Locheck is not yet popular enough to be in `homebrew/core` and we haven't created a tap yet.
```

Comment:
Now that we have this sentence, I would vote for removing "This project is very new and setting up new installation methods takes time." as it could get out of date quickly.